pkgs/development/tools/xc/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib, buildGoModule, fetchFromGitHub, installShellFiles }:
+
+buildGoModule rec {
+  pname = "xc";
+  version = "0.0.154";
+
+  src = fetchFromGitHub {
+    owner = "joerdav";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-GJBSPO0PffGdGAHofd1crEFXJi2xqgd8Vk2/g4ff+E4=";
+  };
+
+  vendorHash = "sha256-XDJdCh6P8ScSvxY55ExKgkgFQqmBaM9fMAjAioEQ0+s=";
+
+  doCheck = false;
```

Comment:
Is there a reason for this? Maybe add a comment here.

Reply:
I used another package as a starting point, should have omitted it, will address!